docs/kit/power-board.md

```diff
@@ -37,11 +37,11 @@ Finally, there are connectors for external Start and On|Off switches. You may co
 | PWR\|FLAT  | Green when powered<br />Flashing red and green when the battery is low                                            | Green                  |
 | 5V         | Green when 5V is being supplied                                                                                   | Green                  |
 | H0–1, L0–3 | Green when the corresponding output is on [^1]<br />Red when the output's current limit is reached                | Off                    |
-| RUN\|ERROR | Orange on power-up, or USB reset <br />Flashing green when ready to run<br /> Solid green when running or booting | Orange                 |
+| RUN\|ERROR | Orange on power-up, or USB disconnection <br />Flashing green when ready to run<br /> Solid green when running or booting | Orange                 |
 
 [^1]: The outputs only turn on when your program runs (specifically, when the `Robot` object is created).
 
-On power-up, the Power Board will emit some beeps, which are related to the version of the firmware it has installed.
+On power-up and while waiting for the start button, the Power Board will emit a beep.
```

Comment:
> while waiting for the start button,

Is this true? This reads to me like board will keep beeping all the while the start button hasn't been pressed

Reply:
8389c8d2884a0a054442004249e42f60addd3fa7